test/test_nested_tensor_functional.py

```diff
@@ -24,78 +24,123 @@ def test_addmm(self):
             [torch.rand(1, 4), torch.rand(1, 4), torch.rand(4, 4)]
         )
 
-    def test_conv2d(self):
+    def test_nn_conv2d(self):
+        inputs = [
+            torch.randn(3, 500, 600),
+            torch.randn(3, 128, 128)
+        ]
+
+        # most of optional params
+        conv2d = torch.nn.Conv2d(3, 33, kernel_size=(3, 5), stride=(2, 1), padding=(4, 2), padding_mode='zeros', dilation=(3, 1), groups=1, bias=True)
```

Comment:
Don't exercise the module but the functional. The module might start using the functional in unexpected ways.

Reply:
add module tests